themes/default/layouts/partials/header.html

```diff
@@ -1,5 +1,5 @@
 {{ if hasPrefix .RelPermalink "/docs/" }}
-    {{ partial "tertiary-nav" . }}
+    {{ partial "top-nav" . }}
```

Comment:
I named this wrong to begin with. (There is another thing called "tertiary nav".) 🤦 Should be right now.